cellprofiler/modules/plugins/measurementtemplate.py

```diff
@@ -148,8 +244,11 @@ def create_settings(self):
     # a template for visible_settings that you can cut and paste here.
     #
     def settings(self):
-        return [self.input_image_name, self.input_object_name,
-                self.radial_degree]
+        return [
+            self.input_image_name,
+            self.input_object_name,
+            self.radial_degree
+        ]
 
     #
     # CellProfiler calls "run" on each image set in your pipeline.
```

Comment:
I noticed you changed these lines in the other template. Should they be changed here too?

Reply:
👍 Done https://github.com/CellProfiler/CellProfiler/pull/3429/commits/2c7c53c7e83a0b7bbac9fba6a02963b58c1accbb